README.md

```diff
@@ -113,6 +113,36 @@ app = Litestar(route_handlers=[hello_world])
 for testing and POC work.
 </details>
 
+## Sponsors
+
+Litestar is an open-source project, and we enjoy the support of our sponsors to help fund the exciting
+work we do.
+
+If you would like to support the work that we do please consider [becoming a sponsor][sponsor-github]
+on [GitHub][sponsor-github] or [Open Collective][sponsor-oc].
+
+We also participate in pledge-based sponsorship with [Polar][sponsor-polar].
+
+Does your company require third-party services offer support contracts?
+We also provide SLAs and support contracts for Litestar.
```

Comment:
Should this be included? In our Github Sponsors, we explicitly state that there is no SLA. Maybe this wording could be changed until we reach a decision on #2861?

Reply:
Ahh yeah good point. I don't want to hold up the sponsor badges going in as its been promised so I can take it out for now.